src/.editorconfig

```diff
@@ -0,0 +1,15 @@
+# editorconfig.org
+root = true
+
+[*]
+indent_style = space
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.{html,md,js,css,sql}]
+indent_size = 2
+
+[*.py]
+indent_size = 4
```

Comment:
@bazzadp any reason not to prefer 2 spaces for consistency with every other language?

Reply:
When I looked at this when setting up the linters I found claims that [4 spaces are wildly more common in Python than two](https://stackoverflow.com/a/1125719).

The linters were therefore set up with the default of 4 spaces so this has been enforced for a while. While pylint can be configured in our config file to be two, it doesn’t look like flake8 can except on command line and the [GitHub super linter doesn’t allow that yet](https://github.com/github/super-linter/issues/987).

One of the things I like about this PR is that it offers the option of enforcing the two standards easily.

Reply:
Also spaces much more important in Python so I actually think it’s a good thing for them to be more obvious there.